<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>إنشاء شكوى عميل</string>
  </property>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>نوع الشكوى</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_create">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
   </property>
   <property name="text">
    <string>إنشاء شكوى</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_complainType">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>120</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="duplicatesEnabled">
    <bool>true</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QComboBox" name="CMB_company"/>
     </item>
     <item>
      <widget class="QComboBox" name="CMB_branch"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>10</y>
      <width>76</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>الشركة</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>الفرع</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>13</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>رقم العميل</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>46</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>اسم  العميل</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_custNo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>13</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_custName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>46</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>رقم الموبايل</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_custPhone">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>80</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>150</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="CMB_section">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>الإداره</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>القسم</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CMB_department">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QDateEdit" name="Qdate_invoice">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>119</width>
      <height>18</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="displayFormat">
     <string>dd-MM-yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2020</year>
      <month>1</month>
      <day>3</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>16</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>رقم الماكينه</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_pos">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>16</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>16</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تاريخ الفاتوره</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>561</width>
     <height>281</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="title">
    <string>بيانات الشكوى </string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>20</y>
      <width>531</width>
      <height>241</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>529</width>
       <height>239</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="details">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
